app/util/data_preparation/jira/prepare-data.py

```diff
@@ -120,8 +121,8 @@ def __get_software_project_keys(jira_api):
     software_project_keys = [project['key'] for project in all_projects if 'software' == project.get('projectTypeKey')]
     if not software_project_keys:
         raise SystemExit("There is no software project in Jira")
-
-    return software_project_keys
+    # Limit number of projects to avoid "Request header is too large" for further requests.
+    return software_project_keys[:PROJECTS_COUNT_LIMIT]
```

Comment:
As far as I remember, there is the same request via the POST method. Maybe we should try to use it?